src/decorators.ts

```diff
@@ -542,4 +543,18 @@ export function OrmEntities(entities: Function[]) {
   return clsdec;
 }
 
-
+export function OperonInitializer() {
+  console.log("Operon Initializer decorator called");
+  function decorator<This, Args extends unknown[], Return>(
+    target: object,
+    propertyKey: string,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    inDescriptor: TypedPropertyDescriptor<(this: This, ctx: InitContext, ...args: Args) => Promise<Return>>)
+  {
+    console.log("Operon Initializer decorator called");
```

Comment:
If we only support one init function, would throw an error here if it's called multiple times.
If we support multiple functions (the current implementation in operon.ts seems to iterate a list), they need to be order independent.

Reply:
We should support multiple.